components/NavigationDrawer/tests/unit/MDCNavigationDrawerScrollViewTests.m

```diff
@@ -544,4 +544,37 @@ - (void)testBottomDrawerTopInset {
   XCTAssertEqualWithAccuracy(fakeHeader.topInset, (CGFloat)7.0, (CGFloat)0.001);
 }
 
+- (void)testExpandToFullScreenAnimationWithFullScreenContent {
+  // Given
+  self.fakeBottomDrawer.trackingScrollView = nil;
+  self.fakeBottomDrawer.originalPresentingViewController.view.bounds = CGRectMake(0, 0, 200, 500);
+  UIViewController *fake = [[UIViewController alloc] init];
```

Comment:
there is already a fake view controller in the set up, do you need to create this?